framework_tvb/tvb/adapters/datatypes/db/spectral.py

```diff
@@ -50,9 +51,12 @@ def fill_from_has_traits(self, datatype):
         # type: (FourierSpectrum)  -> None
         super(FourierSpectrumIndex, self).fill_from_has_traits(datatype)
         self.segment_length = datatype.segment_length
+        self.segment_length = numpy.float64(self.segment_length)
```

Comment:
I would really want to have these casts in a single place, not spread all over the DB entities